iceoryx_posh/test/moduletests/test_roudi_cmd_line_parser.cpp

```diff
@@ -114,7 +118,7 @@ TEST_F(CmdLineParser_test, WrongOptionLeadsUnkownOptionResult)
     args[0] = &appName[0];
     args[1] = &option[0];
 
-    iox::config::CmdLineParser sut;
+    CmdLineParser sut;
     auto result = sut.parse(numberOfArgs, args);
 
     EXPECT_THAT(result.has_error(), Eq(true));
```

Comment:
I know you didn't introduce this code but could you please replace ``EXPECT_THAT`` by ``ASSERT_THAT``? The reason is that if ``result`` doesn't contain an error we face undefined behavior in the next line.

Reply:
Looks like it's done.